api/docs.go

```diff
@@ -1579,6 +1591,10 @@ const docTemplate = `{
                     "type": "string",
                     "readOnly": true
                 },
+                "content_type": {
+                    "description": "Content Type (rpm) of the repository",
```

Comment:
```suggestion
                    "description": "Content Type of the repository  (rpm)",
```
Lets be consistent in placement